### ISSUE TITLE
login as Editor fails repeatedly

### ISSUE THREAD

**Issue by sryu** [yuzoolcode/pulsecms-tracking#53]
I have inconsistent login experience as Editor in v51b4. Sometimes it works, other times it stubbornly refuses to pass. I looked into the login.php and found that 

> #verify password - editor
>    if (    (\pulsecore\wedge\config\get_json_configs()->json->editor_user_enable === true)
>    		and \password_verify($_POST["mpass_pass"], \pulsecore\wedge\config\get_json_configs()->json->editor_user_password)
>  and ($token_check == true)
>     		) 
>

if I disable the $token_check == true line, it works fine. So, guess something is wrong with the part that sets the $token_check for the editor?

**Comment by ghost** [comment]
Thanks for reporting this.
Will take a look and get this into the next round 
5.1B5 will be out today and 5.1 final next week

**Comment by ghost** [comment]
This needs further testing so will be put in the next update

**Comment by sryu** [comment]
So I just installed the 5.1 gold, and did some more test about this issue. 
On my end, this is repeatable: 

1. Open a browser window and log in as admin, do something. 
2. Logout, but do not close the browser window
3. In that same window, try to login as editor. 
This fails to login, and brings up the login window again. I type in passwd for editor, it fails first time, and on second try, works. 

If I try the same, but in step 2, close the browser window, and then open a new window before going to step 3, I don't seem to have the problem. 

Given what I had found earlier about the 
($token_check == true)
issue, it seems that the token_check fails if I try switching from admin to editor in the same browser window. 

(this issue may not arise in ordinary use cases, but while setting up and testing for a client site, it is an irritation.)
